build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -99,6 +99,9 @@ service Execution {
   // * `CANCELLED`: The operation was cancelled by the client. This status is
   //   only possible if the server implements the Operations API CancelOperation
   //   method, and it was called for the current execution.
+  // * `NOT_FOUND`: Due to a transient condition, an unknown operation name,
+  //   or a deletion via the Operations API DeleteOperation method, the
```

Comment:
I don't think we should refer to DeleteOperation like this. Keep in mind that we explicitly state that operations can only be queried using GetOperation, and that no other parts of the Operations API need to be implemented:

https://github.com/bazelbuild/remote-apis/blob/2871680f9f28be4208686bab798fad6c1024d1f1/build/bazel/remote/execution/v2/remote_execution.proto#L74-L79

Noteworthy to mention that Buildbarn doesn't even implement GetOperation, as no sane build client would make use of it. My recommendation for REv3 is that we remove that requirement entirely.

Reply:
"If the server implements" sounds good. Thanks!